package.json

```diff
@@ -10,7 +10,7 @@
     "postinstall": "ngcc --tsconfig './tsconfig.app.json' --use-program-dependencies && ngcc --source 'node_modules/@angular'",
     "ng": "ng",
     "start": "NG_PERSISTENT_BUILD_CACHE=1 ng serve",
-    "build": "NG_PERSISTENT_BUILD_CACHE=1 ng build --configuration production",
+    "build": "ng build --configuration production",
```

Comment:
why we remove this? I meant to remove the config prod thing

Reply:
this has always been prod - how often do we build (rather than server) a dev build?